### ISSUE TITLE
options menu unusable on arcade cabinet

### ISSUE THREAD

**Issue by mcalfin** [ListenerJubatus/smtheme-fiftyOne#22]
Hello! 

I compiled sm5.2 master for my itg2 dedicab the other day and was pleasantly surprised to how beautiful this theme is as the new default. Really like the work that was done here and how modern the design looks. However, the song options menu seems to be unusable without up/down menu buttons on the cabinet. once the options menu loads, select forces the cursor to the top row and start/left/right all begin the song with no way to navigate around. I was able to work around this by changing select to down, but it is not really an ideal solution since most players are used to using select + left/right to change difficulty. It would be really cool if it left/right scrolled up/down the list and pressing start brought up submenus for every option as it does for some, similar to PIU Infinity for example. 

I have no plans to enable the pads as menu buttons, so if that ends up being the suggestion with no plans to fix I suppose I'll just work on fixing it myself for my own local copy, but it would be really nice if this could support native cabinet setups out of the box (especially since stepmania's subtitle does say "Designed for both home and arcade use.")

**Comment by shakesoda** [comment]
the engine is designed for arcade use but that doesn't mean default settings are.

try input options->advanced->menu navigation=three key

if that doesn't fix it 1) it's a bug, 2) try 5.1-new, it definitely works there.

**Comment by mcalfin** [comment]
I did that on the current master branch and it didn't seem to have any effect. I also tried setting options navigation to arcade style as well. i will give 5.1-new a try tonight when i get some time.

![image](https://user-images.githubusercontent.com/12830125/31621653-40afbeac-b268-11e7-98c1-ce819211fe9c.png)
 
Once this menu loads left, right and start all cause the song to start (since play is the first option). left/right do not move it down and neither does start once you've hovering over the other options. instead start brings up the sub-menus and left/right will switch the option (tried all combinations of five/three key and sm/arcade style settings).

will let you know though once i get a chance to test this tonight.

**Comment by ListenerJubatus** [comment]
5.1-new does not use this design; it is a 5.2-specific feature.

![stepmania_2017-12-29_22-15-52](https://user-images.githubusercontent.com/6346433/34451236-dc44b6c6-ece5-11e7-92a6-e53540962300.png)

**Comment by mcalfin** [comment]
Correct, this is only a 5.2 issue. I had briefly looked at it but forgot to come back and comment until I got an email about this thread last night. Only the modifier menu is unaffected by the 3 button setting. If I recall correctly the operator menu functions normal, but since this made it unusable we've just been using simply love instead since I never had time to look at this. We'd prefer not to go back to 5.1-new or 5.0.12 like the rest of the community insists on using, but I would entertain the idea of the default lambda theme if this got fixed. We need USB customs and are using a custom build with coin mode working.

**Comment by ListenerJubatus** [comment]
USB customs got backported to the new 5.1 as well.

The new menu system has a 3-button mode but the config is not exposed in the GUI yet (@kyzentun )
